docs/install/openshift.md

```diff
@@ -0,0 +1,259 @@
+## Requirements
+
+Before proceeding, please ensure that you have an OpenShift cluster running K8s
+1.19+ (OpenShift 4.7+) and have Helm 3.5+ installed. In addition, you'll need to
+install the OpenShift CLI (`oc`) to authenticate to your cluster and create OpenShift
+resources.
+
+You'll also want to install the [latest version of Coder](https://github.com/coder/coder/releases/latest)
+locally in order to log in and manage templates.
+
+## Install Coder with OpenShift
+
+### 1. Authenticate to OpenShift and create a Coder project
+
+Run the following command to login to your OpenShift cluster:
+
+```console
+oc login --token=w4r...04s --server=<cluster-url>
+```
+
+Next, you will run the below command to create a project for Coder:
+
+```console
+oc new-project coder
+```
+
+### 2. Configure SecurityContext values
+
+Depending upon your configured Security Context Constraints (SCC), you'll need to set
+the following `securityContext` values in the Coder Helm chart:
+
+```yaml
+coder:
+  securityContext:
+    runAsNonRoot: true
+    runAsUser: <project-specific UID>
+    runAsGroup: <project-specific GID>
+    readOnlyRootFilesystem: false
+    seccompProfile:
+      type: RuntimeDefault
+    allowPrivilegeEscalation: false
+    capabilities:
+      drop:
+        - ALL
+```
+
+The above values are the Coder defaults. You will need to change these values in
+accordance with the applied SCC. To get a current list of SCCs, run the below command:
+
+```console
+oc get scc
+```
+
+> Note: you must have cluster-admin privileges to manage SCCs
+
+### 3. Set the `CODER_CACHE_DIRECTORY` environment variable
+
+By default, Coder creates the cache directory in `/home/coder/.cache`. Given the
+OpenShift-provided UID, the Coder container does not have permission to write to
+this directory.
+
+To address this issue, you will need to set the `CODER_CACHE_DIRECTORY` environment
+variable in your Helm values to the following:
+
+```yaml
+coder:
+  env:
+    - name: CODER_CACHE_DIRECTORY
+      value: /tmp/coder-cache
+```
+
+### 4. Set access URL, PostgreSQL connection values
```

Comment:
Openshift's "marketplace" offers Postgres as an option which works nicely with Coder. Mark and I left some scratch notes on this [here](https://www.notion.so/coderhq/OpenShift-Notes-a027bc331ee7495193930e037023c14e?pvs=4). I think we should mention this here. Thoughts?

Reply:
@bpmct i think we should - couldn't hurt.